src/main/kotlin/com/nivalit/invoiceninja/sdk/InvoiceApi.kt

```diff
@@ -26,5 +29,32 @@ class InvoiceApi {
             xRequestedWith = InvoiceNinja.xRequestedWith,
             id = id
         ).data
+
+        fun markPaid(id: String) = client.bulkInvoices(
+            X_API_TOKEN = InvoiceNinja.apiToken,
+            xRequestedWith = InvoiceNinja.xRequestedWith,
+            bulkInvoicesRequest = BulkInvoicesRequest("mark_paid", listOf(id))
+        )
+
+        fun send(id: String) =
+//            templateClient.getShowTemplate(
+//                X_API_TOKEN = InvoiceNinja.apiToken,
+//                xRequestedWith = InvoiceNinja.xRequestedWith,
+//                getShowTemplateRequest = GetShowTemplateRequest(
+//                    entity = "invoice",
+//                    entityId = id,
+//                    template = "email_template_invoice"
+//                )
+//            ).body.let{emailBody ->
```

Comment:
This commented code in unnecessary. 
But we can add another function to get preview:
```kotlin
fun preview(id: string) = 
...
```